.github/workflows/test-rc-release.yaml

```diff
@@ -37,9 +104,14 @@ jobs:
           exit 1
 
   create-branch-for-testing-rc-release:
-    needs: validate-manual-input
+    needs: [validate-manual-input, check-rc-testing-annocement]
     runs-on: 'ubuntu-20.04'
-    if: ${{ inputs.issue_url }}
+    if: |
+      always() &&
+      (
+        (github.event_name == 'workflow_dispatch' && inputs.issue_url != '') ||
+        (github.event_name == 'schedule' && needs.check-rc-testing-annocement.outputs.rc_issue_url != '')
```

Comment:
Shouldn't we also add a check here that there is no existing PR in our repo?

Reply:
if we have existing PR, we'll set this value to ''. https://github.com/astronomer/astronomer-providers/pull/1146/files#diff-7cd74bfb9babddfbfa7911a72e5d0784f450360d0a81e9f8319ecba30711b059R88